test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -55,6 +55,33 @@ const loopbackSuccesskMock = RequestMock()
     'access-control-allow-methods': 'POST, OPTIONS'
   });
 
+const loopbackSuccessForChallengeTimeoutLogger = RequestLogger(/introspect|probe|challenge|poll|cancel/, { logRequestBody: true, stringifyRequestBody: true });
+const loopbackSuccessForChallengeTimeoutMock = RequestMock()
+  .onRequestTo(/\/idp\/idx\/introspect/)
+  .respond(identifyWithDeviceProbingLoopback)
+  .onRequestTo(/\/idp\/idx\/authenticators\/poll/)
+  .respond(identifyWithDeviceProbingLoopback)
+  .onRequestTo(/2000\/probe/)
+  .respond(null, 500, {
+    'access-control-allow-origin': '*',
+    'access-control-allow-headers': 'X-Okta-Xsrftoken, Content-Type'
+  })
+  .onRequestTo(/6511\/probe/)
+  .respond(null, 200, {
+    'access-control-allow-origin': '*',
+    'access-control-allow-headers': 'X-Okta-Xsrftoken, Content-Type'
+  })
+  .onRequestTo(/6511\/challenge/)
+  .respond(async (req, res) => {
+    await new Promise((r) => setTimeout(r, 10000));
```

Comment:
can we have a shorter timeout 3100 ?

Reply:
Looks like our test execution is faster (less than 3s), the test is teared down after completion. the long timeout here will not have impact. but I'm going to change it anyway.